src/Uno.UI/UI/Xaml/Controls/TextWrapping.cs

```diff
@@ -7,8 +7,8 @@ namespace Windows.UI.Xaml
 {
 	public enum TextWrapping
 	{
-		NoWrap,
-		Wrap,
-		WrapWholeWords,
+		NoWrap = 1,
```

Comment:
Add note linking to docs: https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.textwrapping?view=winrt-22621